sos/report/plugins/ceph_mgr.py

```diff
@@ -40,16 +40,26 @@ def setup(self):
             "ceph mgr metadata",
         ])
 
-        # more commands to be added later
-        ceph_cmds = [
-            "mgr module ls",
-            "mgr dump",
+        ceph_d_cmds = [
```

Comment:
`ceph_d_cmds`? Did you mean to say `ceph_cmds` or `ceph_mgr_cmds` or something like that?

Reply:
using ceph_d_cmds for ceph_daemon commands list.

Reply:
changed it the way you suggested on IRC - i.e,

                    containers_list[0][1], "ceph daemon %s %s" % (mgrid, cmd))
                    for mgrid in mgr_ids for cmd in ceph_daemon_cmds